src/PrestaShopBundle/Resources/views/Admin/Sell/Order/Order/Blocks/View/details.html.twig

```diff
@@ -36,7 +36,7 @@
 {% endset %}
 
 {% set merchantReturnsTitle %}
-  {{ 'Merchandise returns'|trans({}, 'Admin.Orderscustomers.Feature') }} ({{ orderForViewing.returns.orderReturns|length }})
+  {{ 'Merchandise returns'|trans({}, 'Admin.Orderscustomers.Feature') }} (<span class="count">{{ orderForViewing.returns.orderReturns|length }}</span>)
```

Comment:
I'm not an expert, this does not change how it looks?

Reply:
It depends if the layout have css applied on span

Reply:
By default spans are inline block, same as a raw text, but if some styles are added to span inside another HTMLElement, it could change